mcda/electre/class_assignment.py

```diff
@@ -0,0 +1,203 @@
+from typing import Any, Dict, Union
+
+from mcda.core.aliases import NumericValue
+from mcda.electre.outranking import OutrankingRelation, outranking_relation_marginal
+
+import pandas as pd
+
+
+def assign_tri_rc_class(
+    alternatives: Union[Dict[Any, NumericValue], pd.Series],
+    categories_rank: pd.Series,
+    categories_profiles: pd.Series,
```

Comment:
Categories_rank and categories_profiles can be merged into one argument. I propose something like:
{category1: profile1, category2:profile2,...}
When it comes to Tri-B I something similiar:
{category1: profile1, category2:profile2, .., categoryN:None}
where profileN is boundary profile between categoryN and categoryN+1. Because of that last category wouldn't have any value.
I propose also to order categories ascending or descending and keep that through whole class asigment. Thanks to that there is no need for categories_rank. We can further ensure that user would input correct data in validation.
It can be done also to other class assignments functions.